overlays/builder.py

```diff
@@ -0,0 +1,21 @@
+from subprocess import call
+import os
+import shutil
+import glob
+
+def compile():
+    # The compile needs the include for the spi2 stuff, not enabling that at the moment
+    # as I don't want to pull in the include section from CHIP-dt-overlays
+    call(["dtc", "-O", "dtb", "-o", "overlays/chip-spi2.dtbo", "-b", "o", "-@", "overlays/chip-spi2.dts"])
```

Comment:
In order to compile the SPI and PWM modules, you needed the includes.  I just removed this dependency and re-enabled compiling of the dts on install.

You now need the dtc compiler specified in: "Make a DTBO - Device Tree Overlay Blob" on http://docs.getchip.com/dip.html#development-by-example